test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/NullConditionalExpressions.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@{
+    @ViewBag?.Data
+    @ViewBag.IntIndexer?[0]
+    @ViewBag.StrIndexer?["key"]
+    @ViewBag?.Method(Value?[23]?.More)(SecondValue)?["key"]
+}
+
+@ViewBag?.Data
+@ViewBag.IntIndexer?[0]
+@ViewBag.StrIndexer?["key"]
+@ViewBag?.Method(Value?[23]?.More)(SecondValue)?["key"]
```

Comment:
Why bother testing nested delegates to this extent? It's a corner case that should be visible in at most one or two low-level (`CSharpImplicitExpressionTest`) cases.